pandas/core/frame.py

```diff
@@ -2735,6 +2736,81 @@ def to_parquet(
             **kwargs,
         )
 
+    def to_orc(
+        self,
+        path: FilePathOrBuffer = None,
```

Comment:
Depending on which methods a file-like object needs to implement for to_orc:

`path: FilePath | WriteBuffer[bytes] | None = None,`

Reply:
Done!